src/app/Http/Controllers/SynchronizationController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use App\Enums\PritomnostAbsenceType;
+use App\Models\BusinessTrip;
+use App\Models\PritomnostAbsence;
+use App\Models\PritomnostUser;
+use App\Models\User;
+
+class SynchronizationController extends Controller
+{
+    /**
+     * Synchronize users between Cesty and Pritomnost databases.
+     *
+     * @return void
+     */
+    public function syncUsers(): void
+    {
+        /**
+         * Get all users from the database Pritomnost
+         */
+        $pritomnostUsers = PritomnostUser::all();
+
+        foreach ($pritomnostUsers as $pritomnostUser) {
+            /**
+             * Check if the user exists in the database Cesty
+             */
+            $cestyUser = User::where('personal_id', $pritomnostUser->personal_id)->first();
```

Comment:
Takyto querying cez for-cyklus bude dost pomaly a zatazujuci na DB, hlavne ak sa bude robit pravidelne. Cize spravil by som to cez `LEFT JOIN`, kde `Pritomnost.users` bude na lavej strane, `Cesty.users` na pravej, a ak `Cesty.users` nemaju ekvivalentneho usera, tak tam bude `null`. Potom sa to da preiterovat a robit ify podla toho, ci  tam je `null` alebo nie.